### ISSUE TITLE
weird last-modified behaviour for .json

### ISSUE THREAD

**Issue by tripodsan** [adobe/helix-html-pipeline#31]
```
$ curl -sI https://main--cocacola-journey--hlxsites.hlx.page/helix-config.json
HTTP/2 200
content-type: application/json
last-modified: Thu, 20 Jan 2022 03:54:21 GMT
```

but admin shows:

```json
$ curl https://admin.hlx.page/code/hlxsites/cocacola-journey/main/helix-config.json
{
  "webPath": "/helix-config.json",
  "resourcePath": "/helix-config.json",
  "code": {
    "status": 200,
    "codeBusId": "helix-code-bus/hlxsites/cocacola-journey/main/helix-config.json",
    "permissions": [
      "read",
      "write"
    ],
    "contentType": "application/json",
    "lastModified": "Fri, 01 Apr 2022 16:46:02 GMT",
    "contentLength": "2121",
    "sourceLastModified": "Fri, 01 Apr 2022 16:46:01 GMT",
    "sourceLocation": "https://raw.githubusercontent.com/hlxsites/cocacola-journey/main/helix-config.json"
  },
  ```

and

```json
$ curl https://admin.hlx.page/preview/hlxsites/cocacola-journey/main/metadata.json
{
  "webPath": "/metadata.json",
  "resourcePath": "/metadata.json",
  "preview": {
    "url": "https://main--cocacola-journey--hlxsites.hlx.page/metadata.json",
    "status": 200,
    "contentBusId": "helix-content-bus/9b852c5133df61746fa4cb854124bc1c66f21bb0349ffa50190af423614/preview/metadata.json",
    "contentType": "application/json",
    "lastModified": "Thu, 20 Jan 2022 21:46:24 GMT",
    "sourceLocation": "onedrive:/drives/b!efDI-hf4J0G-a3ALGSF5BBlR37EUliZBgGSrm9jO-GVGOLVenUS8SIJrRErsmsOV/items/016MFFNR7GT2TVEPKU4JAK75RWGB2HEGQD",
    "sourceLastModified": "Thu, 20 Jan 2022 03:54:21 GMT",
```

**Comment by tripodsan** [comment]
i.e. the metadata.json last-modified influences the helix-config.json, which is correct, since the fstab.yaml is included.
however once https://github.com/adobe/helix-admin/issues/380 is fixed, it should only take the last-modified of `fstab.yaml` into account for the non extension-less rendering

**Comment by tripodsan** [comment]
fixed with #33
